packages/flutter/test/widget/scroll_events_test.dart

```diff
@@ -22,6 +23,14 @@ Widget _buildScroller({Key key, List<String> log}) {
 }
 
 void main() {
+  GlobalKey<ScrollableState<Scrollable>> scrollKey;
+
+  Completer<Null> scrollTo(double newScrollOffset, {Duration duration}) {
```

Comment:
We put spaces after the `{` and before the `}` for named argument declarations because we're strange.